notebook2/notebook/notebook.py

```diff
@@ -64,7 +64,7 @@ def read_string(f):
         return f.read().strip()
 
 # Must be int for Kubernetes V1 api timeout_seconds property
-KUBERNETES_TIMEOUT_IN_SECONDS = int(os.environ.get('KUBERNETES_TIMEOUT_IN_SECONDS', 5))
+KUBERNETES_TIMEOUT_IN_SECONDS = float(os.environ.get('KUBERNETES_TIMEOUT_IN_SECONDS', 5))
```

Comment:
this is a reversion to pre-state management pr, since not using timeout_seconds any longer